src/makeRobotAccountant.js

```diff
@@ -7,6 +7,20 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+  const errorCheck = (firstNumber) => {
```

Comment:
You should state what the funcion does in the positive case (what's its general purpose)
```suggestion
  const getSum = (firstNumber) => {
```